nx-X11/programs/Xserver/hw/nxagent/Dialog.c

```diff
@@ -308,7 +305,7 @@ void nxagentLaunchDialog(DialogType dialogType)
               DECODE_DIALOG_TYPE(dialogType), *pid, dialogDisplay);
   #endif
 
-  *dialogDisplay = '\0';
+  dialogDisplay[0] = '\0';
```

Comment:
Functionally equivalent. Just a comment.

Reply:
yeah, but better to read IMHO.